samples/app/src/main/java/com/example/app/env/EnvViewModel.kt

```diff
@@ -13,59 +13,107 @@ import androidx.compose.runtime.mutableStateOf
+        service {
+            authServiceName = "sdkUsernamePasswordJourney"
+        }
+    }
+
+    // Example values for a Ping Advanced Identity Cloud instance
+    val PingAdvancedIdentityCloud = FROptionsBuilder.build {
+        server {
+            url = "https://openam-forgerock-sdks.forgeblocks.com/am"
             realm = "alpha"
-            cookieName = "<cookieName>"
+            cookieName = "29cd7a346b42b42"
             timeout = 50
         }
         oauth {
-            oauthClientId = "<ClientID>"
-            oauthRedirectUri = "<RedirectURI>"
-            oauthCacheSeconds = 0
-            oauthScope = "openid profile email address phone"
-            oauthThresholdSeconds = 0
+            oauthClientId = "sdkPublicClient"
+            oauthRedirectUri = "org.forgerock.demo://oauth2redirect"
+            oauthScope = "openid profile email address"
+            oauthSignOutRedirectUri = "org.forgerock.demo://oauth2redirect"
         }
         service {
-            authServiceName = "Login"
+            authServiceName = "sdkUsernamePasswordJourney"
+        }
+    }
+
+    // Example values for a PingOne instance
+    val PingOne = FROptionsBuilder.build {
+        server {
+            url = "https://auth.pingone.com/3072206d-c6ce-ch15-m0nd-f87e972c7cc3/as"
         }
-        store {
-            // Override the default storage
-            oidcStorage = loadTokenStorage(ContextProvider.context)
-            ssoTokenStorage = loadSSOTokenStorage(ContextProvider.context)
-            cookiesStorage = loadCookiesStorage(ContextProvider.context)
+        oauth {
+            oauthClientId = "6c7eb89a-66e9-ab12-cd34-eeaf795650b2"
+            oauthRedirectUri = "org.forgerock.demo://oauth2redirect"
+            oauthSignOutRedirectUri = "org.forgerock.demo://oauth2redirect"
+            oauthScope = "openid profile email address revoke"
         }
     }
 
-    var current by mutableStateOf(server)
+    // Example values for a PingFederate instance
+    val PingFederate = FROptionsBuilder.build {
+        server {
+            url = "https://pingfed.example.com"
+        }
+        oauth {
+            oauthClientId = "sdkPublicClient"
+            oauthRedirectUri = "org.forgerock.demo://oauth2redirect"
+            oauthSignOutRedirectUri = "org.forgerock.demo://oauth2redirect"
+            oauthScope = "openid profile email address"
+        }
+    }
+
+    var current by mutableStateOf(PingAdvancedIdentityCloud)
         private set
 
     init {
-        servers.add(server)
+        servers.add(PingAM)
+        servers.add(PingAdvancedIdentityCloud)
+        servers.add(PingOne)
+        servers.add(PingFederate)
     }
 
     fun select(context: Context, options: FROptions) {
-        if(options.server.url.contains("pingone")) {
+        // Discover settings from the specified PingOne server's .well-known endpoint
+        if(options.server.url.contains("pingone") || options.server.url.contains("pingfederate")) {
```

Comment:
As PingFed is self-hosted, the chance of the domain containing "pingfederate" is very low.

In addition to the "pingone" check, could we have a separate `if` to check for `.well-known/openid-configuration`, and use `discover` if found?

Reply:
To clarify a bit, in the instructions I'll say something along the lines of:
-  Enter the .well-known URI of your PingFederate server.
If that URL _does not_ contain the word "pingfederate", the sample won't work "out of the box", and I'll need an additional instruction to make sure that "discover" is used.
Hope that makes sense?

Reply:
I have used the instance instead:
```
if(options === PingOne || options === PingFederate) {
```

Reply:
Okay, I'll update the PingFederate instructions to match, cheers Andy.
I see this is heading for `develop`, will it also go to `main`, or only when there's a new release?

Reply:
Only when there is a new release.